alicloud/provider.go

```diff
@@ -261,6 +262,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_common_bandwidth_package_attachment": resourceAliyunCommonBandwidthPackageAttachment(),
 			"alicloud_drds_instance":                       resourceAlicloudDRDSInstance(),
 			"alicloud_elasticsearch_instance":              resourceAlicloudElasticsearch(),
+			"alicloud_cas":                                 resourceAlicloudCas(),
```

Comment:
Please use the whole resource  name `alicloud_cas_certificate`

Reply:
done